test/EndToEnd/NuGet.Config

```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```

Comment:
Does this source support autocomplete? There are tests that will likely fail because of that.

Can we get an exception for this? 
IMO, this doesn't make our tests any better, it just makes them different but same.